### DISCUSSION TITLE
Multiple emailaccounts

### DISCUSSION THREAD

**Discussion by Attacler** [directus/directus#10526] [Feature Requests]
Hi everyone,

Right now the mailservice only supports 1 mailaccount at a time.
Maybe its an idea to make multiple possible so you can send emails from multiple accounts.
For example you can select an emailadres in project settings to send the notifications from.
Also within hooks ect you can use multiple adresses.

Another example is for inside a CRM where you want to make a hook that sends invoices, most organisations use billing@ or something similar to send those emails.

**Comment by Lukas-Sachse** [comment]
I also appreciate this option. In my case all i need are two seperated mail-accounts. One for registration, password-reset and other directus-related things. And the second for sending automatic mails inside custom API hooks.


**Linked to "EMAIL_"-Vars:**
MailService: 
https://github.com/directus/directus/blob/1ded9bdad2a480fb2f6872c3126eb89c6cb27824/api/src/services/mail/index.ts#L27

getMailer()-function: 
https://github.com/directus/directus/blob/1ded9bdad2a480fb2f6872c3126eb89c6cb27824/api/src/mailer.ts#L8

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
